src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -90,7 +90,7 @@ describe('VAOS <DateTimeSelectPage>', () => {
 
     expect(
       screen.getByRole('heading', {
-        level: 2,
+        level: 3,
```

Comment:
This one seemed to consistently fail until I updated it to level 3, which follows the patterns in other tests